### PR TITLE
Update boards.txt

### PR DESCRIPTION
**Pull request by microbotsio**
Update of codecell board parameters

*By completing this PR sufficiently, you help us to review this Pull Request quicker and also help improve the quality of Release Notes*

### Checklist
1. [ ] Please provide specific title of the PR describing the change, including the component name (*eg. „Update of Documentation link on Readme.md“*)
2. [ ] Please provide related links (*eg. Issue which will be closed by this Pull Request*)
3. [ ] Please **update relevant Documentation** if applicable
4. [ ] Please check [Contributing guide](https://docs.espressif.com/projects/arduino-esp32/en/latest/contributing.html)
5. [ ] Please **confirm option to "Allow edits and access to secrets by maintainers"** when opening a Pull Request

*This entire section above can be deleted if all items are checked.*

-----------
## Description of Change
Please describe your proposed Pull Request and it's impact.

## Tests scenarios
Please describe on what Hardware and Software combinations you have tested this Pull Request and how.

(*eg. I have tested my Pull Request on Arduino-esp32 core v2.0.2 with ESP32 and ESP32-S2 Board with this scenario*)

## Related links
Please provide links to related issue, PRs etc.

(*eg. Closes #number of issue*)

### CODE REVIEW

**Comment by P-R-O-C-H-Y** [comment]
@microbotsio Can you please provide better description and title?
Also if you are changing variant to esp32c3, it will not use your [pins_arduino.h](https://github.com/espressif/arduino-esp32/blob/0539ebf322cf9b0620d4a8c357ddc284edc87f64/variants/codecell/pins_arduino.h) file of your variant.

File: boards.txt [hunk 1]
```diff
@@ -41058,43 +41058,158 @@ alfredo-nou3.menu.EraseFlash.all.upload.erase_cmd=-e
...
-codecell.upload.speed=921600
 codecell.serial.disableDTR=false
 codecell.serial.disableRTS=false
 
 codecell.build.tarch=riscv32
 codecell.build.target=esp
 codecell.build.mcu=esp32c3
 codecell.build.core=esp32
-codecell.build.variant=codecell
+codecell.build.variant=esp32c3
```
Comment (me-no-dev) [inline boards.txt:41083]: are you positive of this change?

File: boards.txt [hunk 2]
```diff
@@ -41058,43 +41058,158 @@ alfredo-nou3.menu.EraseFlash.all.upload.erase_cmd=-e
...
 codecell.build.board=ESP32C3_DEV
```
Comment (me-no-dev) [inline boards.txt:41084]: This probably needs to be changed to reflect the board name

**Comment by me-no-dev** [comment]
Please add conscious PR Title and Description

File: boards.txt [hunk 3]
```diff
@@ -41058,43 +41058,158 @@ alfredo-nou3.menu.EraseFlash.all.upload.erase_cmd=-e
...
 codecell.build.bootloader_addr=0x0
-codecell.build.cdc_on_boot=1
+
+codecell.build.cdc_on_boot=0
```
[the remark this answers is not in the document]
Comment (microbotsio) [reply boards.txt:41087]: it is connected to the chip's USB pins

File: boards.txt (hunk 2 above)
Comment (microbotsio) [reply boards.txt:41084]: board name is codecell - will it use codecell varriant pins_arduino.h?

Comment (me-no-dev) [reply boards.txt:41084]: in that case old variant was OK and `build.board` should become `CODECELL`

**Comment by VojtechBartoska** [comment]
any updates on this @microbotsio? Thanks!

**Pull request closed without merging**